src/components/app-root/index.tsx

```diff
@@ -20,11 +22,13 @@ import {ViewPane} from '../view-pane/index';
 
 export interface AppRootProps {
   dataset: Dataset;
+  config: VoyagerConfig;
 }
 
 class AppRootBase extends React.PureComponent<AppRootProps, {}> {
   public render() {
-    const {dataset} = this.props;
+    const {dataset, config} = this.props;
+    const {hideHeader = false, hideFooter = false} = config;
```

Comment:
Given that we already set default somewhere else, maybe the default parameter here is unnecessary?

Reply:
Its just as an extra precaution. I guess I really need to get used to typescript :) If its un-necessary I can remove this.